swagger/v1/openapi.json

```diff
@@ -459,72 +347,46 @@
         "responses": {
           "201": {
             "description": "creates a profile",
-            "examples": {
```

Comment:
There's a bug in rswag that doesn't generate examples. You're supposed to be able to configure it to turn off `--dry-run`, but I haven't been able to get it to work. Instead, you can run the rswag command directly with something like this:

```sh
d-c exec reports-consumer /usr/local/bin/ruby -I/usr/local/bundle/gems/rspec-core-3.9.2/lib:/usr/local/bundle/gems/rspec-support-3.9.3/lib /usr/local/bundle/gems/rspec-core-3.9.2/exe/rspec --pattern spec/requests/\*\*/\*_spec.rb,\ spec/api/\*\*/\*_spec.rb,\ spec/integration/\*\*/\*_spec.rb --format Rswag::Specs::SwaggerFormatter --order defined

```